cms/djangoapps/contentstore/tasks.py

```diff
@@ -190,7 +190,13 @@ def update_search_index(course_id, triggered_time_isoformat):
         CoursewareSearchIndexer.index(modulestore(), course_key, triggered_at=(_parse_time(triggered_time_isoformat)))
 
     except SearchIndexingError as exc:
-        LOGGER.error(u'Search indexing error for complete course %s - %s', course_id, text_type(exc))
+        error_list = exc.error_list
+        LOGGER.error(
+            u"Search indexing error for complete course %s - %s",
```

Comment:
Missing another `%s` for `error_list`, I think this will raise a formatting exception.